src/main.cpp

```diff
@@ -1074,6 +1072,15 @@ bool ContextualCheckTransaction(
                 }
             }
 
+            if (nu6Active) {
+                for (auto it = fundingStreamElements.begin(); it != fundingStreamElements.end(); ++it) {
+                    if (std::holds_alternative<Consensus::Lockbox>(it->first)) {
+                        // TODO: do we want to do something with the lockbox value here?
```

Comment:
Probably not, I think that has to be checked where we check the overall transaction balance.